api/app/signals/apps/sigmax/handler.py

```diff
@@ -26,47 +26,54 @@
 
 SIGNALS_API_BASE = os.getenv('SIGNALS_API_BASE',
                              'https://acc.api.data.amsterdam.nl')
-PLACEHOLDER_STRING = ''
 
 
 class ServiceNotConfigured(Exception):
     pass
 
 
+# TODO SIG-593: implement data mapping if and when it is defined
+
 def _generate_creeer_zaak_lk01_message(signal: Signal):
     """
     Generate XML for Sigmax CreeerZaak_Lk01
     """
-    return CREER_ZAAK.format(
-        PRIMARY_KEY=str(signal.signal_id),
-        OMSCHRIJVING=escape(signal.text),
-        TIJDSTIPBERICHT=escape(_format_datetime(signal.created_at)),
-        STARTDATUM=escape(_format_date(signal.incident_date_start)),
-        REGISTRATIEDATUM=escape(_format_date(signal.created_at)),
-        EINDDATUMGEPLAND=escape(_format_date(signal.incident_date_end)),
-        OPENBARERUIMTENAAM=escape(signal.location.address['openbare_ruimte']),
-        HUISNUMMER=escape(str(signal.location.address['huisnummer'])),
-        POSTCODE=escape(signal.location.address['postcode']),
-        X=escape(str(signal.location.geometrie.x)),
-        Y=escape(str(signal.location.geometrie.y))
-    )
+    # SIGMAX will be set up to receive Signals (meldingen) that have no
+    # address but do have coordinates (middle of park, somewhere on a
+    # body of water, etc.) Here we set the address if we have it.
+    openbare_ruimte = signal.location.address.get('openbare_ruimte', '')
+    huisnummer = signal.location.address.get('huisnummer', '')
+    postcode = signal.location.address.get('postcode', '')
+
+    return render_to_string('sigmax/creeer_zaak_lk01.xml', context={
```

Comment:
As discussed in person. You can just pass the object `signal` as context to the template and use Django template filters and tags to render your template. That's in most cases the more preferred way. 
Just to keep in mind for further iterations.

Reply:
I will.